libraries/AP_Motors/AP_MotorsMatrix.h

```diff
@@ -29,10 +29,13 @@ class AP_MotorsMatrix : public AP_MotorsMulticopter {
     // you must have setup_motors before calling this
     void                set_update_rate(uint16_t speed_hz);
 
-    // output_test - spin a motor at the pwm value specified
+    // output_test_seq - spin a motor at the pwm value specified
     //  motor_seq is the motor's sequence number from 1 to the number of motors on the frame
     //  pwm value is an actual pwm value that will be output, normally in the range of 1000 ~ 2000
-    void                output_test(uint8_t motor_seq, int16_t pwm);
+    virtual void        output_test_seq(uint8_t motor_seq, int16_t pwm) override;
+
+    // set_output - Set a motor output (should only be performed during testing)
```

Comment:
set_output -> output_test_num

I also suggest to make the comment equal in header and implementation (and maybe more consistent with the output_test_seq comment).

Reply:
Done.